### PR TITLE
[GITHUB-2] Fixed config defaults.

### PR DESCRIPTION
**Pull request by rsgoodall**

### CODE REVIEW

File: templates/cloudwatch.yml.j2 [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ---
 region: {{ sansible_prometheus_cloudwatch_exporter.config.region }}
-metrics: {{ sansible_prometheus_cloudwatch_exporter.config.metrics }}
+metrics: 
+{{ sansible_prometheus_cloudwatch_exporter.config.metrics }}
```
Comment (dennisconrad) [inline templates/cloudwatch.yml.j2:4]: Are you sure there is to be a line break?

File: tasks/configure.yml [hunk 2]
```diff
@@ -19,9 +20,11 @@
...
   notify:
     - restart prometheus cloudwatch exporter
+  when: ansible_user_id != "vagrant" and ansible_user_id != "travis"
 
 - name: Prometheus CloudWatch Exporter service start
   become: yes
   service:
     name: prometheus-cloudwatch-exporter
     state: started
+  when: ansible_user_id != "vagrant" and ansible_user_id != "travis"
```
Comment (dennisconrad) [inline tasks/configure.yml:30]: I think it's always more readable like this (line 30, 23, and 11):

```yaml
when:
  - ansible_user_id != "vagrant"
  - ansible_user_id != "travis"
```

File: .travis.yml [hunk 3]
```diff
@@ -12,11 +12,12 @@ branches:
     - master
 
 env:
-  - ANSIBLE_INSTALL_VERSION=2.0.2.0
-  - ANSIBLE_INSTALL_VERSION=2.1.6.0
-  - ANSIBLE_INSTALL_VERSION=2.2.3.0
-  - ANSIBLE_INSTALL_VERSION=2.3.2.0
-  - ANSIBLE_INSTALL_VERSION=2.4.1.0
+  matrix:
```
Comment (dennisconrad) [inline .travis.yml]: Why a matrix here?  It's only trusty with python 2.7, unless I'm missing something.

**Review by dennisconrad — CHANGES_REQUESTED** [review]

File: tasks/configure.yml (hunk 2 above)
Comment (rsgoodall) [reply tasks/configure.yml:30]: I agree it looks more readable, however whenever I've tried it that way it doesn't work.

File: templates/cloudwatch.yml.j2 (hunk 1 above)
Comment (rsgoodall) [reply templates/cloudwatch.yml.j2:4]: Yes, it's to cope with Jinja formatting being rubbish (again).

**Review by dennisconrad — APPROVED** [review]
👍

**Pull request merged**